ironic-config/ironic.conf.j2

```diff
@@ -116,6 +116,8 @@ ramdisk_image_download_source = {{ env.IRONIC_BOOT_ISO_SOURCE }}
 {% endif %}
 {% if env.IRONIC_EXTERNAL_HTTP_URL %}
 external_http_url = {{ env.IRONIC_EXTERNAL_HTTP_URL }}
+{% elif env.IPXE_TLS_SETUP == "true" %}
+external_http_url = https://{{ env.IRONIC_URL_HOST }}:{{ env.IPXE_TLS_PORT }}
```

Comment:
This option is unrelated to iPXE, I'm not quite sure why you change it.

Reply:
Thanks this was a nice catch, I have removed these two lines, these were leftovers from the version where I wanted to do the basic auth differently not via image_info as it has been implemented now.

Reply:
Or maybe from test I don't really remember.